CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ a release.
 ---
 
 ## [Unreleased]
+### Removed
+- Support for `doctrine/orm` < 2.14
```

Comment:
IMO, if we are not dropping support for a major version, we don't need an entry here.
On the other hand, if these changes are intended to fix deprecations, I think we should add an entry in the `Fixed` section what deprecations are fixed.

Reply:
👍 in this case these deprecations are only triggered in tests